### PR TITLE
Migrate from travis to GitHub actions

### PR DESCRIPTION
**Pull request by lojzatran**
Changes:

- I had to add an option to actions/checkout to fetch the whole repo so that git diff works. By default it fetches only 1 commit, thus git diff the and whole linting did not work properly. It is probably better if I could fetch only 2 commits - the current one and the one that should be diff from, but I did not find the way how to do it.
- So the solution is as follows: 
https://github.com/commercetools/k8s-charts/pull/43/files#diff-107e910e9f2ebfb9a741fa10b2aa7100cc1fc4f5f3aca2dfe78b905cbd73c0d2R12

- I also rename folders in `test` from `travis` to `lint` as this is what they are doing.

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: lint
+
+on: push
+
+jobs:
+  lint:
+    name: Lint k8s-charts
```
Comment (JudeNiroshan) [inline .github/workflows/lint.yml:7]: Why do we call it lint? We are also running integration tests on the changed chart.yaml files.

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```
Comment (JudeNiroshan) [inline .github/workflows/lint.yml:11]: can you leave a comment on what does this line do?

**Review by JudeNiroshan — CHANGES_REQUESTED** [review]

**Comment by JudeNiroshan** [comment]
Also mention in the README.md that we are running the lint only considering the last 2 commits. I guess this will be a hidden info after some months.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (lojzatran) [reply .github/workflows/lint.yml:7]: I don't think we run them. Can you point me to the place?

File: .github/workflows/lint.yml (hunk 2 above)
Comment (lojzatran) [reply .github/workflows/lint.yml:11]: I don't think it's useful. This is the very basic of github actions. 
If you don't know what this is doing, you don't know github actions at all and you have to learn github actions. It's like asking for comment on what does `class` keyword means in Java.

**Comment by lojzatran** [comment]
> Also mention in the README.md that we are running the lint only considering the last 2 commits. I guess this will be a hidden info after some months.

We run lint only for the last commit like all the linting we have in other projects. Or do you see any difference here? Because I did not change lint scripts in this PR, maybe you can point me to the place you're talking about.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (JudeNiroshan) [reply .github/workflows/lint.yml:7]: https://github.com/commercetools/k8s-charts/blob/28da58bbc51eaca21ba65ebe2c9804b987e24e7f/.github/workflows/lint.yml#L13

File: .github/workflows/lint.yml (hunk 2 above)
Comment (JudeNiroshan) [reply .github/workflows/lint.yml:11]: I think it's useful. In general, we write a comment when we clone a repo in our bash scripts. The analogy for the `class` in Java is not correct. Even when we write class files in java, we have a documentation section for those classes describing what they are supposed to do. So, a similar idea should be valued in this line. You don't need to explain what the `uses` keyword is for. but mentioning that this step we are cloning the Github repository.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (lojzatran) [reply .github/workflows/lint.yml:7]: This is not integration test. This installs lint stuff for yaml like yamale.

File: .github/workflows/lint.yml (hunk 2 above)
Comment (lojzatran) [reply .github/workflows/lint.yml:11]: https://github.com/commercetools/k8s-charts/pull/43/commits/93746aa15c308f6869d23c93ea1ca8d17a7c0e77

**Review by JudeNiroshan — APPROVED** [review]

**Pull request merged**